### DISCUSSION TITLE
KiCad symbols

### DISCUSSION THREAD

**Discussion by pixelwaster** [dbuezas/lgt8fx#176] [Show and tell]
I've started to learn KiCad and have created a symbol and footprint for the QFP-48 version. This is a square with the pins in their physical location. 32-pin and Dip-20 are coming soon, no date set.

Oddly I found it easier to start the whole process in a text editor rather than the symbol editor.  

Anybody want to create an "LGT kilo" board? Just like the nano only more pins and one of the SI unit left over. Mega, uno, micro, nano, and pico leaves just kilo and milli. 

https://github.com/pixelwaster/LGT8F_Kicad_lib

**Comment by dbuezas** [comment]
What a cool project! We can link to it in the readme when it's ready!

**Comment by pixelwaster** [comment]
Nearly done. Just running into a problem with the SSOP20 part and the -D variant. I do not have the 20 pin part and do not know the actual footprint. There are at least three SSOP-20 footprints. There is nothing in the 1.04 datasheet. Been over then LGT website as well.

If somebody would do me the favor of measuring the length and width of the plastic body and the width from pin edge to pin edge.; or if you have the package outline, send me the information.

The -D variant came in 28, 32, and 48 pin packages I believe. I never saved any D documents and don't have the pinouts or package outlines. Some help with these would also be greatly appreciated. 

Thanks for any help.

**Comment by pixelwaster** [comment]
I do believe it is finished. I've added a pdf with all the x8D and x8P chips I have data for. 
Please check for errors.

[test.pdf](https://github.com/dbuezas/lgt8fx/files/6851719/test.pdf)

**Comment by LaZsolt** [comment]
Your work is unique.
But there are some errors.

> 328D never had 48 pins package
> 328D looks similar, but the internal circuits are differenf from P version.
> 328D are outdated and not in production so only the 32 pins version are available in the market

328D packages
[LGT8Fx8D_packages.pdf](https://github.com/dbuezas/lgt8fx/files/6852892/LGT8Fx8D_packages.pdf)

328D databook (chinese)
[LGT8Fx8D_CH_databook_v1.0.5_25.mar.2016.pdf](https://github.com/dbuezas/lgt8fx/files/6852895/LGT8Fx8D_CH_databook_v1.0.5_25.mar.2016.pdf)

**Reply by pixelwaster** [reply]
Thanks.  The databook I was using was a bit older.
